stdlib/ballerina-http/src/main/ballerina/ballerina/net/http/natives.bal

```diff
@@ -74,6 +74,11 @@ public native function <InRequest req> getQueryParams () (map);
 @Return { value:"The property value" }
 public native function <InRequest req> getProperty (string propertyName) (string);
 
+@Description { value: "Get matrix parameters from the request"}
+@Param { value: "Path to the location of matrix parameters"}
+@Return { value: "A map of matrix paramters which can be found for a given path"}
+public native function <InRequest req> getMatrixParams (string path) (map);
+
```

Comment:
The parameters should be named(i.e: `"path: Path to the location of matrix parameters"`). The receiver should also be documented.

Reply:
*for the specified path